tests/components/homekit/test_type_covers.py

```diff
@@ -31,7 +31,8 @@ def cls(request):
 async def test_garage_door_open_close(hass, cls):
     """Test if accessory and HA are updated accordingly."""
     entity_id = 'cover.garage_door'
-
+    hass.states.async_set(entity_id, None, {})
+    await hass.async_block_till_done()
```

Comment:
If have extracted those changes to #14416, since this might take a bit until it's merged.